bazel/repositories.bzl

```diff
@@ -312,23 +313,36 @@ def _com_github_gcovr_gcovr():
         actual = "@com_github_gcovr_gcovr//:gcovr",
     )
 
+def _com_github_opentracing_opentracing_cpp():
+    location = REPOSITORY_LOCATIONS[
```

Comment:
Please use `_repository_impl()` here so that downstream builders of Envoy can override the dependency definition.

```python
_repository_impl("io_opentracing_cpp")
native.bind(
    name = "opentracing",
    actual = "@io_opentracing_cpp//:opentracing",
)
```

You'll want to adjust the `repository_locations.bzl` name to match.

Reply:
Changed to use _repository_impl and matched names to keys in REPOSITORY_LOCATIONS.